### DISCUSSION TITLE
Coin support request: Siacoin (SC)

### DISCUSSION THREAD

**Discussion by calvintam236** [coinfoundry/miningcore#870] [Ideas]
As there isn't many Siacoin pool out there, and no other pool softs support this coin, I would like to request support for Siacoin (so I can build one).

**Comment by oliverw** [comment]
Implementing Siacoin will certainly become an option once Ethereum support has been implemented. Work on ETH will begin soon.

**Comment by oliverw** [comment]
Postponed indefinitely.
